app/lib/features/space/widgets/non_acter_space_card.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:acter/common/snackbars/custom_msg.dart';
+import 'package:flutter/material.dart';
+import 'package:acter/common/providers/space_providers.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class NonActerSpaceCard extends ConsumerWidget {
+  final String spaceId;
+  const NonActerSpaceCard({super.key, required this.spaceId});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final myMembership = ref.watch(spaceMembershipProvider(spaceId));
+    const fallback = Text(
```

Comment:
I feel like all these plain text strings should be resided in separate class under ```constants``` directory. But right now we can left it as it is till we properly do localisation.

Reply:
+1 for translations ... but until then ...